js/main.js

```diff
@@ -389,7 +393,7 @@ var mapView = {
     switch (menu) {
       case 1:
         var current_user_stats = self.user_data[self.settings.users[user_id].username].stats[0].inventory_item_data.player_stats;
-        $('#subtitle').html('Trainer Info');
+        $('#subtitle').html(self.settings.users[user_id]);
```

Comment:
Can you add the `.username` on line 396?
```
$('#subtitle').html(self.settings.users[user_id]);
```
So it will fetch the username used... and also, on my previously committed code... I've added
```
$("div").find("[data-bot-id='" + self.settings.users[user_id].username + "']").html()
```
Every time the `Info` button is clicked, it will fetch the name of the bot that is currently set (when the bot runs `UpdateLiveStats`) on the top of the bot menus (not the username)...